UI/Authentication.OidcDemo/README.md

```diff
@@ -0,0 +1,14 @@
+# WebAuthenticationBroker with OpenID Connect  (OIDC)
+
+This application demonstrates the usage  of the `WebAuthenticationBroker` in Uno with an OpenID Connect endpoint.
+
+The library used for the authentication  is [`IdentityModel.OidcClient`](https://github.com/IdentityModel/IdentityModel.OidcClient), a project backed by the [.NET  foundation](https://dotnetfoundation.org/projects/identitymodel). You can find the documentation [here](https://identitymodel.readthedocs.io/en/latest/native/overview.html).
+
+Important: because browser  restrictions (popup blockers), opening a window must be done during the handling  of the event of a user interaction. So it's preferable to use the *manual  mode* instead of the automatic one and prepare the starting URL in  advance. If you don't care or if you're not planning to use WebAssembly, you can  use the *automatic  mode* and implement it directly the [WAB  Browser](https://identitymodel.readthedocs.io/en/latest/native/overview.html).
```

Comment:
```suggestion
Important: because of browser restrictions (popup blockers), opening a window must be done during the handling  of the event of a user interaction. So it's preferable to use the *manual  mode* instead of the automatic one and prepare the starting URL in  advance. If you don't care or if you're not planning to use WebAssembly, you can use the *automatic  mode* and implement it directly with the [WAB  Browser](https://identitymodel.readthedocs.io/en/latest/native/overview.html).
```